lib/common.css

```diff
@@ -197,3 +197,13 @@ button.goog-buttonset-default:hover {
   color: <<FORE_SECONDARY>>;
 }
 /** Update End **/
+
+/** Fixes Playlist button **/
+.material .sj-play-button-0 #buttonContent.sj-play-button {
+    background: <<FORE_SECONDARY>>;
+}
+
+.material .sj-play-button-0 #pulse.sj-play-button {
+    background: <<FORE_SECONDARY>>;
+    opacity: .3;
```

Comment:
@MarshallOfSound sort of unrelated, but isn't that regex incorrectly not escaping the `.`, which would end up matching anything in that space?